ports/verticaldropheroeshd/port.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 3,
+    "name": "verticaldropheroeshd.zip",
+    "items": [
+        "VerticalDropHeroesHD.sh",
+        "verticaldropheroeshd"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Vertical Drop Heroes HD",
+        "porter": [
+            "Zehmaluco"
+        ],
+        "desc": "Vertical Drop Heroes is a procedural platformer RPG hybrid with roguelike elements, where your hero adventures through randomly generated stages. Armed with a variety of skills and traits, you must use your cunning and reflexes to survive the dangers of each stage.",
+        "desc_md": null,
+        "inst": "Get the game on Steam or GOG, place game files on the ports/verticaldropheroeshd/gamedata/ directory.",
+        "inst_md": "Get the game on Steam or GOG, place game files on the ports/verticaldropheroeshd/gamedata/ directory.",
```

Comment:
You dont have to fill in the Markdown versions when your not using Markdown features. Just regular will do.

Reply:
You still have double instructions, wont break anything so its fine :)